ci/scripts/run-e2e-test.sh

```diff
@@ -71,6 +71,8 @@ cluster_stop
 echo "--- e2e, $mode, batch"
 RUST_LOG="info,risingwave_stream=info,risingwave_batch=info,risingwave_storage=info" \
 cluster_start
+python3 ./e2e_test/ddl/alter_function_set_schema.py &
```

Comment:
Now we can start udf server inside slt file as follows:
```
system ok
python3 e2e_test/udf/test.py &
```

How about reuse `e2e_test/udf/test.py` and move function related altering test to `udf` directory. Currently the tests under `ddl` directory will be integrated into deterministic test, I'm afraid that madsim can't handle it for now.